extensions/authy/src/component/OtpList.tsx

```diff
@@ -100,10 +176,28 @@ export function OtpList(props: { isLogin: boolean | undefined; setLogin: (login:
         <List.Item
           key={index}
           title={item.name}
-          subtitle={`${item.otp}`}
+          accessoryTitle={`${item.otp}`}
+          subtitle={`${item.issuer || item.accountType}`}
```

Comment:
Here is a problem with it. Native Authy apps don't have either issuer or account type. 
<img width="726" alt="image" src="https://user-images.githubusercontent.com/13866979/148258680-b11da2ea-2cf0-43d7-aeb2-e9d320f43341.png">
Also, a lot of my accounts don't have issuer and have type authenificator 
<img width="743" alt="image" src="https://user-images.githubusercontent.com/13866979/148259081-f6347cb3-64a6-44bc-b9c0-cfeaa13a8b09.png">
I guess it better to show just an issuesr if it exisits

Reply:
I've updated this now so you shouldn't get `undefined` or `authenticator` as you're right these are not useful, can you check and see if you get anything else that looks out of place there?